_meeting-notes/2019-11-26.md

```diff
@@ -0,0 +1,148 @@
+- ==@stebalien==
+  - Improving output and exiting with a non-zero exit code on failure:
+      - https://github.com/ipfs/testground/pull/181
+      - Unrelated? test failure: https://travis-ci.com/ipfs/testground/jobs/259682338
+          - Works locally.
+  - Jitter etc:
+      - https://github.com/ipfs/testground/pull/172
+      - Should be ready for review.
+  - Dropping inbound connections
+      - WIP
+- ==@jimpick==
+  - SHIPPED
+    - Packer + Ansible + Terraform https://github.com/ipfs/testground/pull/111
+  - Ready for PR
+    - Infra: Setup worker nodes automatically on boot https://github.com/ipfs/testground/pull/175
+- ==@nonsense==
+  - Started work on minimal daemon (https://github.com/ipfs/testground/issues/149) and submitted initial/draft PR (https://github.com/ipfs/testground/pull/183).
+
+
+## :question: Questions (Blockers, Technical Design needs)
+> We only have 30 mins for this section, use your time wisely
+
+- (@daviddias) What's the state of SideCar?
+  - status
+    - patch for jitter 
+    - WIP patch for filtering inbound connections
+    - next subnet support
+    - sidecar gets installed through the makefile (not done yet)
+    - Will it work in macOS?
+      - Yes
+    - Should we have Vagrant to develop locally?
+      - Doesn't seem necessary yet
+    - Any help needed?
+      - Review the jitter PR?
+        - check https://github.com/ipfs/testground/blob/master/plans/dht/test/set-up.go#L26-L44 how it works
+      - Does anyone want to own the makefile?
+        - Yes -> Jim! <3 AI: do this 
+- (@daviddias) What's the state of libp2p instrumentation?
+  - AFAWK no work has gotten done on this
+  - Some ideation happening (context vs internal logger) but none set in stone.
+  - Jim has some old code (how old?). Revamping might take ~3 weeks
+  - We will be using Uber Zap.
+- (@nonsense) How do we name the project? Is it `TestGround` or `Test Ground` or `Testground`? I suggest we always use the first or the third, just so that we are consistent in code and docs.
+  - @daviddias - I've been using TestGround & Test Ground but converging to `TestGround` <- the people like this one :)
+  - for search purposes it is better to be one word.
+- (@nonsense) Location of this call? Link at the top? Calendar entry differs from Hackmd link...
+  - 
+- (@jimpick) Can we get a quick explanation of what the 'daemon' is?
+  - AI(Anton): Write or link to where the daemon is documented
+  - AI(??): Spec needs to be revisited asap
+    - AI(Jim): Redline the spec in the areas that it is clearly out of date
+- (@stebalien) Multiple libp2p versions?
+  - No work done yet
+  - How implement this?
+    - Multiple Docker containers
+    - Have multiple build steps 
+  - How would one do this?
+    - A) `testground build a && testground build b && testground run a+b`
+    - B) Change the manifest?
+    - C) Have it described in the main.go of the plan?
+  - AI(Steven): Create an issue describing this design and a potential proposal
```

Comment:
Putting a 📍 here so that I can link to this part of the notes.

Reply:
For reference, @jimpick brought the ask to have this documented back in October 29 too https://github.com/ipfs/testground/pull/89/files#r340507309. @jimpick did you ever got the chance to write something down about it?

Reply:
Ah fun, Oct 22 as well, this has been a recurring one! :)

<img width="673" alt="image" src="https://user-images.githubusercontent.com/1211152/70086182-bbd47080-1609-11ea-93a2-7f3361b6c04e.png">